src/app/containers/StoryPromo/index.stories.jsx

```diff
@@ -58,11 +59,13 @@ storiesOf('Containers|Story Promo/Canonical', module)
   .add('canonical', () => getCanonicalStoryPromo(firstFixture))
   .add('audio promo', () => getCanonicalStoryPromo(audioFixture))
   .add('video promo', () => getCanonicalStoryPromo(videoFixture))
-  .add('standard link promo', () => getCanonicalStoryPromo(standardPromo));
+  .add('standard link promo', () => getCanonicalStoryPromo(standardPromo))
+  .add('video link promo', () => getCanonicalStoryPromo(videoPromo));
```

Comment:
I’m just a bit confused on the naming conventions here? On line 61 it’s adding a 'video promo' but it’s getting a videoFixture. But on line 63 it’s called a 'video link promo' but you’re getting a videoPromo?

Reply:
I agree with Harvey here, I think the strings should be changed to ```audio fixture``` and ```video fixture```. These functions are only ever used in this file, I think it's worth changing since it's only in this file.

Reply:
Thanks @HarveyPeachey @lokh01 , I renamed these stories for more clarity.